### ISSUE TITLE
Use document.ready or equivalent

### ISSUE THREAD

**Issue by spaceninja** [spaceninja/date-list#9]
from @vosechu:

> No `$(document).ready({})` or equivalent?

**Comment by spaceninja** [comment]
Wow, this turns out to be a surprisingly complex topic.

https://www.html5rocks.com/en/tutorials/speed/script-loading/

TL;DR: The simplest way to ensure your scripts load in order without blocking the rendering of the page is to stick them at the very end of the document, as I've done here.

I can enhance this by adding `defer` to the scripts, which will allow parallel downloading, but since my script relies on the CDN scripts and does a `document.write`, I can't use `async`.

As for document.ready, there's a vanilla JS replacement (`DOMContentLoaded`), but it's not needed if the scripts are loaded last.

https://www.sitepoint.com/jquery-document-ready-plain-javascript/

So — I guess the takeaway is that `document.ready` isn't needed for a simple page like this with scripts loaded last, but `defer` might help speed things up by allowing parallel downloading.

**Comment by spaceninja** [comment]
Actually, looking at my network tab, it looks like even plain script elements allow parallel downloading now, so browsers must have improved since my ancient jQuery days.

So `defer` would only benefit me if I wanted to move my scripts to the `head`, which I don't feel any particular need to do.

**Comment by spaceninja** [comment]
Well, nevermind pagespeedtest says I should defer anyway. I'll re-open.

https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fdates.oscorp.net%2F
